exercises/exercise-13/database.ts

```diff
@@ -1,13 +1,85 @@
-export class Database<T> {
+import * as fs from 'fs';
```

Comment:
Exercise 13 was probably more work than all the others put together. I wonder if it could be split?